src/parser_TEST.cc

```diff
@@ -59,6 +62,30 @@ sdf::SDFPtr InitSDF()
   return sdf;
 }
 
+/////////////////////////////////////////////////
+/// Checks emitted warnings for custom/unknown elements in log file
+TEST(Parser, CustomUnknownElements)
+{
+  std::string pathBase = PROJECT_SOURCE_PATH;
+  pathBase += "/test/sdf";
+  const std::string path = pathBase +"/custom_and_unknown_elements.sdf";
+
+  sdf::SDFPtr sdf = InitSDF();
+  EXPECT_TRUE(sdf::readFile(path, sdf));
+
+  std::string line;
+  std::string pathHome = getenv("HOME");
+  std::ifstream file(pathHome + "/.sdformat/sdformat.log");
+
+  if (file.is_open())
+  {
+    std::getline(file, line);
+    file.close();
+  }
+
+  EXPECT_NE(line.find("XML Element[test_unknown]"), std::string::npos);
```

Comment:
Can you also check that `test:custom` is not in the file?

Reply:
95500e0e75f75a1e5ab69e4b264138cbe78f477a